src-ghc/Pact/Server/Server.hs

```diff
@@ -54,6 +55,9 @@ data Config = Config {
 instance ToJSON Config where toJSON = lensyToJSON 1
 instance FromJSON Config where parseJSON = lensyParseJSON 1
 
+type ServerSetup = (HistoryChannel, InboundPactChan, (String -> IO ()), Int, FilePath)
+type ServerThreads = (Async (), Async ())
+
```

Comment:
I think we need to factor this differently to avoid the type synonyms, which amongst other things expose a bunch of random parameters needed to call `runApiServer`. Instead, make `setupServer` just return `(Async,Async,IO ())` with the last being the command to invoke `runApiServer`.